history-of-the-quakers-v3/original/32-book-6-chapter-12.adoc

```diff
@@ -443,12 +464,17 @@ but the preface, written by Thomas Trafford, and the postscript, by William Edmu
 the former explaining the motives and authority of the friends in their concern,
 and the latter briefly recapitulating the subject,
 I think not unworthy of introducing in this place,
-as a specimen of the sense and judgment of Our friends of this age and place.
+as a specimen of the sense and judgment of our friends of this age and place.
 "`Love not the world, neither the things that are in the world:
 If any man love the world, the love of the Father is not in him.`" 1 John 2:15.
 
-=== PREFACE.
+[.embedded-content-document.epistle]
+--
+
+[.old-style]
```

Comment:
I wasn't sure how to so these "preface" and "postscript". I didn't think it should have the normal postscript label, but I could be wrong. There was no body to this epistle. And letter headings didn't seem right either.

Reply:
yup, i'm not sure what i would do here either.  it looked good in the PDF so let's go with it. 👍